scripts/in_container/run_resource_check.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import psutil
+from rich.console import Console
+
+console = Console(force_terminal=True, color_system="standard", width=180)
+
+
+def get_size(kilobytes):
+    """
+    Convert kilobytes into gigabytes
+    1 Gigabytes = 1,048,576 Kb
+    """
+    factor = 1000000000
+    value_gb = kilobytes // factor
+    return value_gb
+
+
+def resoure_check():
+    """
+    Use gsutil to get resources in kylobytes: memory, cpus and disk
+    """
+    resources = {}
+    print("\nChecking resources.\n")
+
+    # Memory available
+    svmem = psutil.virtual_memory()
+    mem_available = get_size(svmem.total)
+    resources.setdefault('Memory', []).append(mem_available)
+
+    # Cpus available
+    cpus_available = psutil.cpu_count(logical=True)
+    resources.setdefault('Cpus', []).append(cpus_available)
+
+    # Disk: Get all disk partitions /
+    partitions = psutil.disk_partitions()
+    partition_usage = psutil.disk_usage(partitions[0].mountpoint)
+    disk_available = get_size(partition_usage.free)
+    resources.setdefault('Disk', []).append(disk_available)
+
+    return resources
+
+
+def resoure_validate():
+    resources = resoure_check()
+    warning_resources = False
+    check = "OK"
+
+    resources.setdefault("Memory", []).append(4)
+    resources.setdefault("Cpus", []).append(2)
+    resources.setdefault("Disk", []).append(20)
+
+    for resource, available in resources.items():
+
+        check = '' if resource == "Cpus" else 'GB'
+
+        if (
+            resource == "Memory"
+            and available[0] < 4
+            or resource == "Cpus"
+            and available[0] < 2
+            or resource == "Disk"
+            and available[0] < 20
+        ):
+            console.print(f"[yellow]WARNING!!!: Not enough {resource} available for Docker.")
+            print(f"At least {available[1]}{check} of {resource} required. You have {available[0]}{check}\n")
+            warning_resources = True
+        else:
+            console.print(f" * {resource} available {available[0]}{check}. [green]OK.\n")
```

Comment:
```suggestion
            console.print(f" * {resource} available {available[0]}{check}. [green]OK.")
```

Reply:
NIT. It will look nicer without separate line:

![image](https://user-images.githubusercontent.com/595491/149023045-6a8e3f44-2a25-4ff6-a1f1-c084c9913d64.png)

Reply:
Done :), definitely looks great without space.
![Screenshot from 2022-01-12 10-08-00](https://user-images.githubusercontent.com/58795858/149166520-6516d0fc-b6ee-40bb-aeea-28b312a457a4.png)